### DISCUSSION TITLE
Ebusd Adapter, Homeassistant, mqtt topics (ebusd/700/*) gone?

### DISCUSSION THREAD

**Discussion by cpolzer** [john30/ebusd#1067] [Q&A]
Hello,

I realized 2 days ago, that I don't recieve any data for the ebusd/700/ topic and sub topics anymore. 

Along with that, it seams the heater and hot water stopped working the same time those messages disappeared. 

What i had done was trying to write a message to the  ebusd/700/HwcSFMode to set it to load mode (aka manually enable the water heating). I did so by writing the json value to the topic.

Beforehand, everything was running fine for month

Can it be that i somehow caused a problem on the ebus? 
Is there any chance i can fix this myself? Or is it just bad coincidence?

Thanks in advance!

P.s: My heater is a Vaillant Arotherm VWL 85/3 with a unitower and a recovair 360.

edit: typos

**Comment by cpolzer** [comment]
Edit: After reflashing the ebus adapter, everything bus wise is back. Not sure what happened here , but i cleared the topic from the retained messages and now the mqtt 700 bus messages are back.

**Reply by haarkop** [reply]
hello, 
can you send me what the config looks like at you and where you have the files at my problem with 720 and ecot tec plus stove

**Reply by haarkop** [reply]
![image](https://github.com/john30/ebusd/assets/85073788/10bbeaa4-fdc8-47b1-8207-fb6f9cca6fa2)
